src/PublicAPI.Shipped.txt

```diff
@@ -19,8 +19,6 @@ Microsoft.Azure.StackExchangeRedis.AzureCacheOptions.ThrowOnTokenRefreshFailure
 Microsoft.Azure.StackExchangeRedis.IAzureCacheTokenEvents
 Microsoft.Azure.StackExchangeRedis.IAzureCacheTokenEvents.ConnectionReauthenticated -> System.EventHandler<string!>?
 Microsoft.Azure.StackExchangeRedis.IAzureCacheTokenEvents.ConnectionReauthenticationFailed -> System.EventHandler<Microsoft.Azure.StackExchangeRedis.ConnectionReauthenticationFailedEventArgs!>?
-Microsoft.Azure.StackExchangeRedis.IAzureCacheTokenEvents.TokenRefreshed -> System.EventHandler<Microsoft.Identity.Client.AuthenticationResult!>?
```

Comment:
Removing lines from this file means we're making a breaking change, so we need to make sure we increment the major version of the package to 2.0.0. 
Update in: Directory.Build.props
Add "<Version>1.1.0</Version>" in Microsoft.Azure.StackExchangeRedis.csproj